calibrators/docs/tutorials/mapping_based_calibrator.md

```diff
@@ -0,0 +1,162 @@
+# mapping_based_calibrator
+
+In this tutorial, we present a hands-on tutorial of the `mapping_based_calibrator`, particularly its lidar-lidar calibration capabilities. Although we provide a pre-recorded rosbag, the flow of the tutorial is meant to show the user the steps they must perform in their own use cases with live sensors.
+
+General documentation regarding this calibrator can be found [here](../../mapping_based_calibrator/README.md).
+
+## Setup
+
+This tutorial assumes that the user has already built the calibration tools.
+Installation instructions can be found [here](../../../README.md).
+
+## Data preparation
+
+Please download the data (rosbag) from [here](https://drive.google.com/drive/folders/1_pGds5HDyt8C55sD2qEH4TJMGzIOCCyV).
+
+The rosbag includes four pointcloud topics published by different lidar sensors and also includes `/tf_static` information.
+
+Note that in the real-world environment, the user should record a rosbag with all of the lidar topics and tf topics that are needed, which should be similar to the provided bag. Afterward, the user could start driving the vehicle to collect the data in the rosbag and then calibrate sensors with the recorded rosbag.
+
+## Environment preparation
+
+### Overall calibration environment
+
+The required space for calibration depends on the vehicle and sensors used. It is recommended to have a large enough space for the vehicle
+to drive around 30 to 50 meters. It is also recommended to ensure that the environment is rich in natural landmarks suitable for registration-based mapping in all directions. This will help the lidar capture sufficient details beyond simple features like lane surfaces or walls.
+
+### Vehicle
+
+Before starting the calibration, the user needs to drive the vehicle to collect the pointcloud for building the map. While recording data during the experiment, slow down the vehicle's speed as much as possible. For instance, driving slower than 2 km/hr is good for recording quality data. Also, during the experiment, try to avoid people walking around the vehicle and keep the surroundings static.
+
+## Launching the tool
+
+In this tutorial, we use the RDV of Tier IV (R&D Vehicle).
+First, run the sensor calibration manager:
+
+```text
+ros2 run sensor_calibration_manager sensor_calibration_manager
+```
+
+In `project`, select `rdv`, and in `calibrator`, select `mapping_based_calibrator`. Then, press `Continue`.
```

Comment:
Continue is with capital letters but project is not
(this also applies to other places I think)

Reply:
Thanks, fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/b24df517e7cd059b3bfa2caa5601cb4f8dc69323